Guia para resolver conflictos

Clarissa0512 opened the pull request:
## Issues

_¿Qué se está resolviendo?_
- Guía para resolver conflicto.
## Todo
_¿Qué queda por hacer?_
- Revisar
## Áreas impactadas
_¿Qué páginas o componentes están afectados?_
- resolver-conflictos.md

Clarissa0512 committed:
Subiendo guia para resolver conflictos

Clarissa0512 committed:
Guía terminada

Clarissa0512 committed:
añadiendo yarn.lock y node_modules/

cosola commented on resolver-conflictos.md:
```diff
@@ -0,0 +1,28 @@
+#Resolviendo conflictos
+
+Esta guia es para resolver los conflictos con Git.
```
Sugiero cambiar como sigue:
Esta guia es para resolver conflictos con Git.

cosola commented on resolver-conflictos.md:
```diff
@@ -0,0 +1,28 @@
+
+Esta guia es para resolver los conflictos con Git.
+
+NOTA: Estos conflictos ocurren cuando existen choques entre el contenido de tu rama y el contenido de Master, como Git no sabe que contenido tomar, el tuyo o el de master, de ja que seas tú que decidas cual debe tomar.
```
Cambios sugeridos:
NOTA: Conflictos ocurren cuando existen choques entre el contenido de tu rama y el contenido de Master. Como Git no sabe que contenido escoger, el tuyo o el de Master, deja que seas tú, o alguien con autoridad, quien decida cual contenido aceptar.

cosola commented on resolver-conflictos.md:
```diff
@@ -0,0 +1,28 @@
+
+NOTA: Estos conflictos ocurren cuando existen choques entre el contenido de tu rama y el contenido de Master, como Git no sabe que contenido tomar, el tuyo o el de master, de ja que seas tú que decidas cual debe tomar.
+
+## 1. Ver el archivo que tiene el conflicto, si tiene el conflicto contendrá la siguiente estructura:
```
Falta claridad aquí; no puedo sugerir.

cosola commented on resolver-conflictos.md:
```diff
@@ -0,0 +1,28 @@
+
+>>>>>>> Master
+
+## 2. Debes borrar lo que no quieres, si quieres los cambios de Master borras los cambios de tu rama y si quieres los de tu rama, entonces borra los de Master.
```
Sugerencia:

## 2. Debes borrar cambios que no quieres mantener. Si quieres los cambios de Master, borras los cambios de tu rama; si quieres los de tu rama, entonces borra los de Master.

cosola commented on resolver-conflictos.md:
````diff
@@ -0,0 +1,28 @@
+
+## 2. Debes borrar lo que no quieres, si quieres los cambios de Master borras los cambios de tu rama y si quieres los de tu rama, entonces borra los de Master.
+
+## 3. Luego de que arreglas los conflictos, el archivo está listo para ser añadido: usa ```git add.
````
Sugerencia:
## 3. Luego de resolver los conflictos, el archivo está listo para ser añadido.  Usa ```git add.

Clarissa0512 committed:
Removiendo "los" en la descripción de la guia

Aqui remuevo los de la descripción 'Esta guia es para resolver los conflictos con Git.'

Clarissa0512 committed:
Modificando mensaje de la NOTA de Git

Aquí modifico el contenido del mensaje que contiene la NOTA.

Clarissa0512 committed:
Aquí cambio el contenido del paso #2 de la guia para resolver conflictos con Git

Aquí cambio la oración del segundo paso para resolver conflictos con Git.

Clarissa0512 committed:
Mejorando el tercer paso de resolver conflictos con Git

Aquí estoy mejorando el tercer paso de resolver conflictos con Git

Clarissa0512 committed:
Arreglando ejemplo de como se ve un archivo con conflicto

Aquí arreglo la muestra de como se ve un archivo con conflictos.

Clarissa0512 committed:
Arreglando los comentarios en la guia para reslover conflictos

Aquí arreglo los comentarios en la guia para reslover conflictos que no funcionaban correctamente.

Clarissa0512 committed:
Añadiendo comillas a las instrucciones de la guia

Aquí añado comillas a las instrucciones de la guia en los pasos 3, 4 y 5.
